smokeTest.js

```diff
@@ -0,0 +1,112 @@
+process.argv.forEach((val, index) => {
+    console.log(`${index}: ${val}`);
+});
+  
+const XETHRU_STATE = require('./SessionStateXethruEnum.js')
+const MOV_THRESHOLD = 17
+const IM21_DOOR_STATUS = require('./IM21DoorStatusEnum')
+const testLocation1Id = 'TestLocation1'
+const door_coreID = 'door_coreID'
+const radar_coreID = 'radar_coreID'
+const { helpers } = require('brave-alert-lib')
+
+function getRandomInt(min, max) {
+    min = Math.ceil(min)
+    max = Math.floor(max)
+    return Math.floor(Math.random() * (max - min) + min) //The maximum is exclusive and the minimum is inclusive
+}
+
+function getRandomArbitrary(min, max) {
+    return Math.random() * (max - min) + min
+}
+
+
+let destinationURL = process.argv[2];
+let recipientNumber = process.argv[3];
+let twilioNumber = process.argv[4];
+
+
+  
+const axios = require('axios').default;
```

Comment:
I think I removed the dependency on axios in https://app.clickup.com/t/j6yuzk. You'll probably need to put it back.

Reply:
Hmmm, did you not need to do this? I don't see it in `package.json`.